crates/threads-xform/src/lib.rs

```diff
@@ -442,7 +472,7 @@ fn inject_destroy(
         },
     );
 
-    let destroy_id = builder.finish(Vec::new(), &mut module.funcs);
+    let destroy_id = builder.finish(vec![tls_base, stack_alloc, stack_size], &mut module.funcs);
```

Comment:
I only added the last parameter, the first two locals were just not assigned.
So I guess `__wbindgen_thread_destroy()` didn't work until now when called for another thread.